src/Nethermind/Nethermind.Synchronization/SnapSync/SnapSyncFeed.cs

```diff
@@ -110,90 +110,93 @@ public override SyncResponseHandlingResult HandleResponse(SnapSyncBatch? batch,
         }
 
         public SyncResponseHandlingResult AnalyzeResponsePerPeer(AddRangeResult result, PeerInfo peer)
+{
```

Comment:
Indentation is still wrong.

Reply:
Sorry for the delayed response. 
I guess now Indentations are managed.